src/libpgexporter/management.c

```diff
@@ -319,7 +319,6 @@ pgexporter_management_write_status(int socket)
 {
    configuration_t* config;
 
-   pgexporter_start_logging();
```

Comment:
I removed these as in all situations (in main), logging was started before this function is run, and sometime after this function is finished.

Starting and closing logging inside creates the problems:

1. fclose(log_file) had no check for `log_file == NULL` (I added this).
2. `pgexporter_stop_logging` is called in main again later, which leads to double free (or closing of logging prematurely, just because the user made a `status`/`details` request using `pgexporter-cli`).

Because of these, I removed logging from these functions.

Reply:
We need to investigate how we can do logging in `cli.c` as well...